File: modelo/Oleada.java
package modelo;

public class Oleada {
    private int idOleada;
    private int cantidadEnemigos;
    private String tipoEnemigo;
    private int vidaBase;
    private int velocidadBase;

    public Oleada(int idOleada, int cantidadEnemigos, String tipoEnemigo, int vidaBase, int velocidadBase) {
        this.idOleada = idOleada;
        this.cantidadEnemigos = cantidadEnemigos;
        this.tipoEnemigo = tipoEnemigo;
        this.vidaBase = vidaBase;
        this.velocidadBase = velocidadBase;
    }

    public int getIdOleada() { return idOleada; }
    public int getCantidadEnemigos() { return cantidadEnemigos; }
    public String getTipoEnemigo() { return tipoEnemigo; }
    public int getVidaBase() { return vidaBase; }
    public int getVelocidadBase() { return velocidadBase; }

    @Override
    public String toString() {
        return "Oleada [ID=" + idOleada + ", Cantidad=" + cantidadEnemigos +
               ", Tipo=" + tipoEnemigo + ", VidaBase=" + vidaBase + ", VelocidadBase=" + velocidadBase + "]";
    }
}

File: Python/modelo/oleada.py
class Oleada:
    def __init__(self, id_oleada, cantidad_enemigos, tipo_enemigo, vida_base, velocidad_base):
        self.id_oleada = id_oleada
        self.cantidad_enemigos = cantidad_enemigos
        self.tipo_enemigo = tipo_enemigo
        self.vida_base = vida_base
        self.velocidad_base = velocidad_base

    def __str__(self):
        return (f"Oleada [ID={self.id_oleada}, Cantidad={self.cantidad_enemigos}, "
                f"Tipo={self.tipo_enemigo}, VidaBase={self.vida_base}, "
                f"VelocidadBase={self.velocidad_base}]")